apply/default_applier.go

```diff
@@ -49,19 +51,23 @@ type DefaultApplier struct {
 type ActionName string
 
 const (
-	PullIfNotExist ActionName = "PullIfNotExist"
-	MountRootfs    ActionName = "MountRootfs"
-	UnMountRootfs  ActionName = "UnMountRootfs"
-	MountImage     ActionName = "MountImage"
-	Config         ActionName = "Config"
-	UnMountImage   ActionName = "UnMountImage"
-	Init           ActionName = "Init"
-	Upgrade        ActionName = "Upgrade"
-	ApplyMasters   ActionName = "ApplyMasters"
-	ApplyNodes     ActionName = "ApplyNodes"
-	Guest          ActionName = "Guest"
-	Reset          ActionName = "Reset"
-	CleanFS        ActionName = "CleanFS"
+	PullIfNotExist            ActionName = "PullIfNotExist"
+	MountRootfs               ActionName = "MountRootfs"
+	UnMountRootfs             ActionName = "UnMountRootfs"
+	MountImage                ActionName = "MountImage"
+	Config                    ActionName = "Config"
+	UnMountImage              ActionName = "UnMountImage"
+	Init                      ActionName = "Init"
+	Upgrade                   ActionName = "Upgrade"
+	ApplyMasters              ActionName = "ApplyMasters"
+	ApplyNodes                ActionName = "ApplyNodes"
+	Guest                     ActionName = "Guest"
+	Reset                     ActionName = "Reset"
+	CleanFS                   ActionName = "CleanFS"
+	PluginConfig              ActionName = "PluginConfig"
```

Comment:
rename to DumpPlugins is better I think.

Reply:
Ok